packages/viewer/src/utils/components/FileInput.vue

```diff
@@ -120,17 +121,21 @@ const emits = defineEmits<{
 
 // Reactive data
 const inputLocalFile = useTemplateRef<HTMLInputElement>('inputLocalFile')
-const internalInvalidMessage = ref<string>('')
 
 // Computed properties
-const filePathInfo = computed(() =>
-    model.value ? `${model.value.name}, ${model.value.size / 1000} kb` : ''
-)
 const maxFileSizeHuman = computed(() => humanFileSize(maxFileSize))
 
-// Methods
+// Create a writable computed ref for the model to match the expected type
+const modelRef = computed({
+    get: () => model.value,
+    set: (value: File | undefined) => {
+        model.value = value
+    },
+})
```

Comment:
a ref/computed of a model seems a bit overkill, are you sure this is required?

Reply:
Yes, you are right. I can modify the useFieldValidation and other components, so we don't need to do this "hack". Thanks!